config/v1/types_securityprofile.go

```diff
@@ -0,0 +1,304 @@
+package v1
+
+// SecurityProfile defines the schema for a security profile. This object is used
+// by operators to apply network security settings to operands.
+// +union
+type SecurityProfile struct {
```

Comment:
This looks TLS specific.  Why are we being generic in the name?

Reply:
It's also server specific.  Also, it's incredibly specific to the Mozilla settings - not generic to other frameworks.

Reply:
Do you prefer `TLSProfile`, `NetworkConnectionProfile`, `NetworkSecurityProfile`, `SecureConnectionProfile` or do you have a recommendation?